CSharp/DevVmPowershell/DevVmPsModules/BaseModule.cs

```diff
@@ -53,7 +55,7 @@ protected override void ProcessRecord()
 				WriteVerbose($"Start - {nameof(ProcessRecord)} method");
 
 				base.ProcessRecord();
-				ProcessRecordCode();
+				RetryProcessRecordCode(ProcessRecordCode, TimeSpan.FromSeconds(5), 5);

```

Comment:
Can we move that constant integer arguments to the constants class. So that its easy to change in the future.

Reply:
Updated, please re-review